src/payment/data/service.js

```diff
@@ -78,17 +79,17 @@ export async function postCoupon(code) {
 }
 
 export async function deleteCoupon(id) {
-  const { data } = await App.apiClient
-    .delete(`${App.config.ECOMMERCE_BASE_URL}/bff/payment/v0/vouchers/${id}`)
+  const { data } = await getAuthenticatedHttpClient()
+    .delete(`${getConfig().ECOMMERCE_BASE_URL}/bff/payment/v0/vouchers/${id}`)
     .catch(handleBasketApiError);
   return transformResults(data);
 }
 
 export async function getDiscountData(courseKey) {
-  const { data } = await App.apiClient.get(
-    `${App.config.LMS_BASE_URL}/api/discounts/course/${courseKey}`,
+  const { data } = await getAuthenticatedHttpClient().get(
+    `${getConfig().LMS_BASE_URL}/api/discounts/course/${courseKey}`,
     {
-      xhrFields: { withCredentials: true },
+      withCredentials: true,
```

Comment:
Sanity check - this `xhrFields` thing was unnecessary?  Or was it wrong?

Reply:
It's jQuery syntax.